rocketpy/Environment.py

```diff
@@ -3221,6 +3221,117 @@ def allInfo(self):
+            speedOfSound=[self.speedOfSound(i) for i in grid],
+            density=[self.density(i) for i in grid],
+            windVelX=[self.windVelocityX(i) for i in grid],
+            windVelY=[self.windVelocityY(i) for i in grid],
+            pressure=[self.pressure(i) / 100 for i in grid],
+            temperature=[self.temperature(i) for i in grid],
+        )
+        if self.atmosphericModelType != "Ensemble":
+            return plotInfo
+        currentMember = self.ensembleMember
+        # List for each ensemble
+        plotInfo["ensembleWindVelocityX"] = []
+        for i in range(self.numEnsembleMembers):
+            self.selectEnsembleMember(i)
+            plotInfo["ensembleWindVelocityX"].append(
+                [self.windVelocityX(i) for i in grid]
+            )
+        plotInfo["ensembleWindVelocityY"] = []
+        for i in range(self.numEnsembleMembers):
+            self.selectEnsembleMember(i)
+            plotInfo["ensembleWindVelocityY"].append(
+                [self.windVelocityY(i) for i in grid]
+            )
+        plotInfo["ensembleWindSpeed"] = []
+        for i in range(self.numEnsembleMembers):
+            self.selectEnsembleMember(i)
+            plotInfo["ensembleWindSpeed"].append([self.windSpeed(i) for i in grid])
+        plotInfo["ensembleWindDirection"] = []
+        for i in range(self.numEnsembleMembers):
+            self.selectEnsembleMember(i)
+            plotInfo["ensembleWindDirection"].append(
+                [self.windDirection(i) for i in grid]
+            )
+        plotInfo["ensemblePressure"] = []
+        for i in range(self.numEnsembleMembers):
+            self.selectEnsembleMember(i)
+            plotInfo["ensemblePressure"].append([self.pressure(i) for i in grid])
+        plotInfo["ensembleTemperature"] = []
+        for i in range(self.numEnsembleMembers):
+            self.selectEnsembleMember(i)
+            plotInfo["ensembleTemperature"].append([self.temperature(i) for i in grid])
+
+        # Clean up
+        self.selectEnsembleMember(currentMember)
+        return plotInfo
+
+    def allInfoReturned(self):
+        """Returns as dicts all data available about the Environment.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        ------
+        info: Dict
+            Information relevant about the Environment class.
+        """
+
+        # Dictionary creation, if not commented follows the SI
+        info = dict(
+            grav=self.g,
+            launch_rail_length=self.rL,
+            elevation=self.elevation,
+            modelType=self.atmosphericModelType,
+            modelTypeMaxExpectedHeight=self.maxExpectedHeight,
+            windSpeed=self.windSpeed(self.elevation),
+            windDirection=self.windDirection(self.elevation),
+            windHeading=self.windHeading(self.elevation),
+            surfacePressure=self.pressure(self.elevation) / 100,  # in hPa
```

Comment:
I am not really sure departing from SI will do us any good 😟

Reply:
This has the same explanation as the previous question about ```self.pressure(i) /100``` and the solution will be the same, implement in FE and maintain ```rocketpy``` as SI